daprdocs/content/en/operations/troubleshooting/common_issues.md

```diff
@@ -195,3 +195,10 @@ The following example shows how to set the Host IP env var to `127.0.0.1`:
 ```bash
 export DAPR_HOST_IP=127.0.0.1
 ```
+## None of my components are getting loaded when my application starts. I keep getting "Error component X cannot be found"
+
+This is usually due to one of the following issues
+
+- You may have defined the `NAMESPACE` variable locally or deployed your components into a different namespace in Kubernetes. Check which namespace your app and the components are deployed to.
```

Comment:
If this is a k8s specific issue, probably should add that in the Q title

Reply:
No, this is both local and K8s. In fact the reason I added this, was because I set NAMESPACE="XXX" and everything stopped working!